lib/tether/interfaces.go

```diff
@@ -35,6 +36,7 @@ type Operations interface {
 	SetupFirewall() error
 	Apply(endpoint *NetworkEndpoint) error
 	MountLabel(ctx context.Context, label, target string) error
+	MountFileSystem(ctx context.Context, source url.URL, target string, mountOptions string) error
```

Comment:
The name is redundant.  You can only mount filesystems.  How about `MountTarget`?  Open to suggestions.  But MountFilesystem sounds redundant.